Makefile

```diff
@@ -1,21 +1,21 @@
-.PHONY: build-example check clean fmt
+.PHONY: build-integration-tests check clean fmt
 
 clean:
 	cargo clean
 
 fmt:
-	cargo +nightly fmt --all
+	cargo fmt --all
```

Comment:
IMO We should keep nightly for rust fmt.

It looks like we didn't copy over our rustfmt rules to this repo yet, but most of them break without nightly. A lot of them are very nice to have too.